test/cli_run_memory_test.go

```diff
@@ -164,6 +164,59 @@ func (suite *PouchRunMemorySuite) TestRunWithMemoryswappiness(c *check.C) {
 	c.Assert(out, check.Equals, memSwappiness)
 }
 
+// TestRunWithMemoryReservation is to verify the valid running container
+// with memory-reservation
+func (suite *PouchRunMemorySuite) TestRunWithMemoryReservation(c *check.C) {
+	SkipIfFalse(c, environment.IsMemoryReservationSupport)
+
+	// test run with invalid memory reservation
+	cname := "TestRunWithMemoryReservationInvalid"
+	res := command.PouchRun("run", "-d",
+		"-m", "500m",
```

Comment:
could we use less memory here? According to the testing machine, I am not sure that it will impact the case.